cmd/olfullnode/save_state.go

```diff
@@ -619,3 +654,63 @@ func DumpGovProposalsToFile(pm *governance.ProposalMasterStore, writer io.Writer
 		})
 	}
 }
+
+func DumpNetworkDelegatorsToFile(nd *network_delegation.Store, writer io.Writer, fn func(writer io.Writer, obj interface{}) bool) {
+	prefixList := []network_delegation.DelegationPrefixType{network_delegation.ActiveType, network_delegation.MatureType, network_delegation.PendingType}
+	delimiter := ","
+	version := nd.State.Version()
+
+	for _, prefix := range prefixList {
+		iterator := 0
+		startList(writer, prefix.GetJSONPrefix())
```

Comment:
I noticed that the json.Marshal can write object/slice with internal delimiters automatically for us. I am too lazy to manually write these delimiters in my loop. LOL